sormas-ui/src/main/java/de/symeda/sormas/ui/events/eventLink/EventSelectionField.java

```diff
@@ -101,11 +106,27 @@ public EventSelectionField(EventDto event, Set<String> excludedUuids, boolean se
 		initializeGrid();
 	}
 
+	public EventSelectionField(EventGroupReferenceDto eventGroupReference, Set<String> excludedUuids) {
+		this.searchField = new TextField();
+		this.infoPickOrCreateEvent = I18nProperties.getString(Strings.infoPickOrCreateSuperordinateEventForEvent);
```

Comment:
I guess that's not the correct I18n property because it references superordinate events